src/components/Lesson/Lesson.js

```diff
@@ -42,10 +42,10 @@ const LessonButton = ({path, lessons, t}) => {
 
 const onclickAndSetCheckboxes = (path, checkboxes, setCheckbox) => {
   const labels = document.getElementsByTagName('label');
```

Comment:
Problemet er at `document.getElementsByTagName()` returnerer en HTMLCollection (https://developer.mozilla.org/en-US/docs/Web/API/HTMLCollection), som ikke er en array.

I stedet for å gjøre om for-løkkene, foreslår jeg heller at du konverterer HTMLCollection til en array, f.eks. med
```
const labels = Array.from(document.getElementsByTagName('label'));
```
(Du kunne også brukt det litt kortere, men mer kryptiske
```
const labels = [...document.getElementsByTagName('label')];
```
, men det er kanskje mest lesbart med `Array.from()`? Du kan bestemme du.)